views.py

```diff
@@ -0,0 +1,15 @@
+"""
+This view serves only the basic website structure like index page
+"""
+from django.shortcuts import render
+
+
+def index(request):
+    return render(request, 'index.html')
+def about_us(request):
```

Comment:
you should stick to pip8 and leave 2 blank lines between functions

Reply:
> you should stick to pip8 and leave 2 blank lines between functions

He means PEP8.

Reply:
yes